### DISCUSSION TITLE
i can't to search anything

### DISCUSSION THREAD

**Discussion by ghost** [community/community#11956] [Other Feature Feedback, Questions, & Ideas]
when i search a keyword , the cs.github.com return me a this :
`Search failed: search failed`

what should i do???

**Comment by colinwm** [comment]
Is this still happening? Search should be working for everyone now

**Reply by shaozhihao031** [reply]
but the response to my application is "You’re already on the waitlist for GitHub Code Search! We’ll email you when we’ve enabled it on your account. Make sure your [primary email address is up-to-date](https://github.com/settings/emails)."

**Comment by dunetrader** [comment]
Hello, I am experiencing the same issue - Every time I search it results with a "Search Failed" text. Kindly, let me know the reason for the same. Is there a search limit I have crossed? any document describing the limit

**Comment by cachius** [comment]
@colinwm Is it possible that the changes related to cs.github.com somehow interfere with the regular code search? The same queries I ran three days ago don't yield any results now. E.g. searching for  [RegisterURLSchemeAsSupportingFetchAPI](https://github.com/chromium/chromium/search?q=RegisterURLSchemeAsSupportingFetchAPI) in chromium/chromium should lead among others to [render_thread_impl.cc](https://github.com/chromium/chromium/blob/main/content/renderer/render_thread_impl.cc). However:
>We couldn’t find any code matching 'RegisterURLSchemeAsSupportingFetchAPI' in [chromium/chromium](https://github.com/chromium/chromium)
You could [search all of GitHub](https://github.com/search?q=RegisterURLSchemeAsSupportingFetchAPI) or try an [advanced search](https://github.com/search/advanced?q=RegisterURLSchemeAsSupportingFetchAPI).

Same behavior wether logged in or out. 'Go to File' still works fine.
